frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -249,6 +240,18 @@ const VerticalBlock = (props: BlockPropsWithoutWidth): ReactElement => {
       )
     }) !== undefined
 
+  useEffect(() => {
+    if (wrapperElement.current) {
+      observer.observe(wrapperElement.current)
+    }
+    return () => {
+      observer.disconnect()
+    }
+    // We need to update the observer whenever the scrolling is activated or deactivated
+    // Otherwise, it still tries to measure the width of the old wrapper element.
+    /* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Wouldn't it be better to perform this when `wrapperElement.current` changes? That's more explicitly when we need to update the observer (and may not need the disable the eslint)?

Reply:
Yeah, that was my first version. But react throws a warning/error that you shouldn't current of a ref in dependencies. It did seem to work, but it sounded a bit discouraging. Also see here: https://stackoverflow.com/a/60476525

Reply:
I think it works fine in this specific case since we are not dependent on the rendering triggered by the change of current. But not sure if it is fine to use this "anti-pattern" or the alternative with `activateScrollToBottom`.

Reply:
Boo. My other idea was to have the memo re execute when `activateScrollToBottom` is called, but it's not that strong.

Reply:
ahh yes, adding it to `useMemo` would also work, but I think it's almost the same effect since the changed observer will trigger the effect as well. I think I will leave it as is for now.